en/asgardeo/mkdocs.yml

```diff
@@ -480,7 +480,9 @@ nav:
       - Organization APIs: apis/organization-apis/index.md
       - Get access for organization APIs: apis/organization-apis/authentication.md
       - API resource management API: apis/organization-apis/api-resource-management-rest-api.md
-      - Application management API: apis/organization-apis/org-application-management.md
+      - Application management API: 
+        - Application management API (Shared Applications): apis/organization-apis/shared-application-management.md
+        - Application management API: apis/organization-apis/organization-application-mgt.md
```

Comment:
Can this cause existing paths (apis/organization-apis/org-application-management) to break? if yes, we may need to add redirects.

Reply:
Added the redirection